packages/workbox-expiration/src/models/CacheTimestampsModel.ts

```diff
@@ -76,7 +78,9 @@ class CacheTimestampsModel {
 
     // Previous versions of `workbox-expiration` used `this._cacheName`
     // as the IDBDatabase name.
-    deleteDatabase(this._cacheName);
+    if (this?._cacheName) {
```

Comment:
I'm curious as to why it's `this?` instead of `this`. When would `this` not be defined?

Reply:
Glad you asked, I meant to ask you about that, we call the method to upgradeDB in tests, for example: https://github.com/GoogleChrome/workbox/blob/v6/test/workbox-expiration/sw/test-CacheTimestampsModel.mjs#L16, and that is when `this` would not be defined. I am not sure if this is still the correct way to do this tests, because the test would be the one creating the IDB because it is opened before each test, maybe we should refactor the tests to open the DB only in the test cases that needs it and only by passing the name?

Reply:
Turns out my approach also wouldn't work, so I sort of separated the dbupgrade and the deleting of old dbs and that seems to work, let me know what you think. Thank you!